spec/features/user_adds_coverage_to_a_course_spec.rb

```diff
@@ -1,21 +1,27 @@
 require "rails_helper"
 
 feature "user adds coverage to a course" do
-  scenario "successfully" do
+  scenario "successfully", js: true do
     subject = create(:subject)
     course = create(:course)
     outcome = create(:outcome, course: course)
     user = user_with_assessments_access_to(course.department)
 
     visit manage_assessments_course_path(course, as: user)
     click_on t('manage_assessments.courses.show.add_a_class')
-    select subject.title, from: "Subject"
-    select outcome.nickname, from: "Outcome"
+    selectize subject.title, from: "Subject"
+    selectize outcome.nickname, from: "Outcome"
     click_button t('helpers.submit.coverage.create')
 
     within("#matched_outcomes") do
       expect(page).to have_content(subject.title)
       expect(page).to have_content(outcome.nickname)
     end
   end
+
+  def selectize(item, from:)
```

Comment:
I looked into this a bit, and I think the only things that would change would be the 2nd and 3rd lines of this method -- we'd be able to replace those with `page.driver.execute_script` executing some javascript to set the value of the selectize. It doesn't look like it'd be much simpler than what we have here and what we have here is a bit more true to how a user would interact with it. So I think we should go with this for now. Can revisit if this gives us problems.